specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/products/{productId}/reviews/{reviewId}/comments": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get the comments of a given review",
+        "operationId": "ReviewComment_GetComments",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetReviewComments.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "reviewId",
+            "in": "path",
+            "required": true,
+            "description": "Review Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "skipToken",
+            "in": "query",
+            "description": "Skip token to continue getting comments from a given point",
+            "x-nullable": true
+          },
+          {
+            "type": "integer",
+            "name": "pageSize",
+            "in": "query",
+            "description": "maximum number of comments to return",
+            "format": "int32",
+            "default": 50,
+            "x-nullable": false
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "default": "2023-01-01-preview",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of comments for the given review",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewCommentDto"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "description": "In case request structure is wrong",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "500": {
+            "x-nullable": false,
+            "description": "In case an error has occurred in the server",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PageResultOfReviewDto": {
+      "type": "object",
+      "required": [
+        "count"
+      ],
+      "properties": {
+        "items": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReviewDto"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "nextPageLink": {
```

Comment:
This needs to be `nextLink` not `nextPageLink`

Reply:
Done